sdk/workflow/workflow.go

```diff
@@ -212,6 +229,38 @@ func (workflow *ConductorWorkflow) StartWorkflowsAndMonitorExecution(startWorkfl
 	return workflow.executor.MonitorExecution(workflowId)
 }
 
+func (workflow *ConductorWorkflow) AddWorkflowTags(workflowName string, tags map[string]string) error {
```

Comment:
after reviewing this again, I realized this is not aligned with the `ConductorWorkflow` api. We already have `workflow.name` and this is just delegating to `workflow.executor.AddWorkflowTags(workflowName, tags)`